src/Pages/List/Components/Pagination.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+const Pagination = ({
+  roomsPerPage,
+  totalRooms,
+  setCurrentPage,
+  currentPage,
+  goToAnotherPage,
+}) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalRooms / roomsPerPage); i++) {
+    pageNumbers.push(i);
+  }

```

Comment:
```suggestion
  const pageNumbers = new Array(Math.ceil(totalRooms / roomsPerPage)).fill(1).map((el,idx)=>idx+1);
```

for 문으로 절차적으로 도는 프로그래밍 하는 것 보다 method들 잘 활용해서 새로운 배열 만드시는게 가독성, 안정성 측면에서 더 좋습니다!
저 코드 동작 이해안되시면 질문주세요!

Reply:
총 방의 갯수를 한 페이지당 방 수로 나눈 값을 올림처리하고,  생성된 어레이의 요소 수 만큼 일단 1로 채운 다음~~뱅글뱅글돌면서 1씩 올려준다! 맞나요?

Reply:
- 총 방의 갯수를 한 페이지당 방 수로 나눈 값을 올림처리하고, 
- 올림처리한 만큼의 length를 가진 새로운 Array를 만들고 
- 생성된 어레이의 요소 수 만큼 일단 1로 채운 다음
(new Array로 만들면 empty value로만 채워져있어서 map을 돌 때 순회가 안되어서 일단 임의값으로 넣어준 부분입니다)
- 뱅글뱅글돌면서 idx + 1 한 값으로 다시 만들어준다(인덱스는 0부터 시작하니까 1부터 시작하기 위해서 +1 해주는 부분입니다)